pkg/handlers/update.go

```diff
@@ -58,6 +58,14 @@ func AppUpdateCheck(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// update last updated at time
```

Comment:
this doesn't look like the right place for this.  we should update the time stamp after a successful request to the replicated.app server later in this handler.

Reply:
Yeah I wasn't entirely sure where exactly this should go. Initially it was further down but maybe it was too far and so it would only be set if there were updates available but it should also update the timestamp if there are no updates.